### PR TITLE
Section on --no-titlebar for Emacs 29

### PR DESCRIPTION
**Pull request by licht1stein**

### CODE REVIEW

File: README.org [hunk 1]
```diff
@@ -111,8 +111,8 @@ By default =emacs-plus@29= uses the following features.
 
 *** Options
 
-| Option                    | Description                                                                  |
-|---------------------------+------------------------------------------------------------------------------|
+| Option                  | Description                                                                  |
+|-------------------------+------------------------------------------------------------------------------|
```
Comment (d12frosted) [inline README.org:115]: Your alignment ignores `=` symbols :) 

image

File: README.org [hunk 2]
```diff
@@ -121,9 +121,12 @@ By default =emacs-plus@29= uses the following features.
 | =--with-x11=              | build with x11 support                                                       |
 | =--with-xwidgets=         | build [[#xwidgets-webkit][→ with xwidgets]] support                                                |
 | =--without-cocoa=         | build a non-Cocoa version of Emacs (terminal only)                           |
-| =--with-imagemagick=      | build with =imagemagick= support                                             |
+| =--with-imagemagick=      | build with =imagemagick= support                                               |
 | =--with-native-comp=      | build with native compilation aka [[#gccemacs][→ gccemacs]]                                 |
 
+*** No Title Bar
+Please note, that ~--with-no-titlebar~ is no longer needed in Emacs 29, since the same can be achieved natively using [[https://github.com/d12frosted/homebrew-emacs-plus#no-titlebar][this method]].
```
[the remark this answers is not in the document]
Comment (licht1stein) [reply README.org]: Did it

File: README.org (hunk 1 above)
Comment (licht1stein) [reply README.org]: Removed the alignment, sorry 🤦‍♂️

**Comment by licht1stein** [comment]
@d12frosted resolved everything and added a header

**Review by d12frosted — APPROVED** [review]
💯 дякую :)

**Pull request merged**